app/com/gu/contentapi/sanity/package.scala

```diff
@@ -23,10 +23,11 @@ package object sanity extends ScalaFutures with Matchers with IntegrationPatienc
   def incrementIncidentCount = {
     incidentCount += 1
     lastIncidentDateTime = Some(DateTime.now)
+    println(s"Incident count: $incidentCount")
```

Comment:
Fair enough, let's leave it in then. We can add a proper logger, e.g. Logback, in another PR.